opendevin/events/action/commands.py

```diff
@@ -10,6 +10,7 @@
 class CmdRunAction(Action):
     command: str
     thought: str = ''
+    timeout: int = 120
```

Comment:
Is this the same timeout as the sandbox_config.timeout? If it's the same, can we use that value as default, rather than defining another default?

Reply:
Yeah.. I wanted to, yet i was not sure how to get that value consistently from the config which is only loaded at runtime..